doc/Command-Reference.md

```diff
@@ -5258,6 +5262,17 @@ The "fec-stats" subcommand is used to disply the interface fec related statistic
   Ethernet16        U           0             0                 0    1.77e-20       0.00e+00
   ```
 
+The "trim" subcommand is used to display the interface packet trimming related statistic.
+
+- Example:
+  ```
+  admin@sonic:~$ show interfaces counters trim
+        IFACE    STATE     TRIM
```

Comment:
We could add TRIM_PKTS as this will be only packet counter

Reply:
@kperumalbfn done